### ISSUE TITLE
subtitle in device-chat

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#1225]
currently, the subtitle in the device-chat shows the fake-email-address device@localhost

i suggest to replace this, by the string `device_talk_subtitle` (reads sth. as "locally generated messages")

also the text "Messaging disabled (this is the device chat)" that shows instead of the input field look a bit weird to me, at least the explanation in brackets. maybe just hide the input bar and show nothing (this is what was done on android+ios)

**Comment by Simon-Laux** [comment]
> also the text "Messaging disabled (this is the device chat)" that shows instead of the input field look a bit weird to me, at least the explanation in brackets. maybe just hide the input bar and show nothing (this is what was done on android+ios)

This is standard for desktop, as its the same disabled composer when leaving a group, I believe hiding it is more confusing.

**Comment by r10s** [comment]
okay, then maybe reword the text, i did a suggestion at https://github.com/deltachat/deltachat-desktop/pull/1231 - i think for now this would do the job.
